tardis/model/tests/test_base.py

```diff
@@ -202,3 +204,48 @@ def test_ascii_reader_exponential_law():
     for i, mdens in enumerate(expected_densites):
         assert_almost_equal(model.density[i].value, mdens)
         assert model.density[i].unit ==  u.Unit(expected_unit)
+
+###
+# Save and Load
+###
+
+
+@pytest.fixture(scope="module")
+def hdf_file_path(tmpdir_factory):
+    path = tmpdir_factory.mktemp('hdf_buffer').join('model.hdf')
+    return str(path)
+
+
+@pytest.fixture(scope="module")
+def actual_model():
+    filename = 'tardis_configv1_verysimple.yml'
+    config = Configuration.from_yaml(data_path(filename))
+    model = Radial1DModel.from_config(config)
+    return model
+
+
+@pytest.fixture(scope="module", autouse=True)
+def to_hdf_buffer(hdf_file_path, actual_model):
```

Comment:
@yeganer If I return `hdf buffer path` from this function , then it same as input argument `hdf_file_path`. I think , If I return `hdf_file_path` here , then it is inconsistent .